Makefile

```diff
@@ -6,10 +6,12 @@ all:
 lint:
 	$(MAKE) -C tiny86 lint
 	$(MAKE) -C mttn lint
+	verible-verilog-lint tiny86/*.v
```

Comment:
Nice job using the linter as well! If we're pulling in the dep might as well use it for sure. You'll see in the `tiny86/Makefile` we have some linting already, but the more coverage the better.

Reply:
Note: this should now do like `$(MAKE) -C tiny86 lint` to expose that Makefile target to the root of the repo. Same with formatting.